tests/neg/i4044a.scala

```diff
@@ -8,7 +8,7 @@ def test(given QuoteContext) = {
     a // error
     $a
     '{$a} // error
-    '{ given QuoteContext = ???; '{$a} } // error
+    '{ given QuoteContext =???; '{$a} } // error
```

Comment:
Why? Same for `tests/neg/i4044b.scala`

Reply:
@nicolasstucki It's regex replacement breakages. There's simply too much to change - 2000 lines already! It's even too much to just inspect every line. 

Can you take this over, to fix the remaining problems? I am running out of time here.

Reply:
Will fix them